src/libraries/System.Runtime.InteropServices/gen/ComInterfaceGenerator/GeneratedComInterfaceAttributeAnalyzer.cs

```diff
@@ -0,0 +1,87 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Collections.Immutable;
+using System.Diagnostics.CodeAnalysis;
+using System.Runtime.InteropServices;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.Diagnostics;
+
+#nullable enable
+namespace Microsoft.Interop
+{
+    [DiagnosticAnalyzer(LanguageNames.CSharp)]
+    public class GeneratedComInterfaceAttributeAnalyzer : DiagnosticAnalyzer
```

Comment:
Should we put analyzers in an Analyzers subfolder and namespace? That is what we did for `LibraryImportGenerator` and I liked the separation/organization.

Reply:
I like that too, I've move the related files to an Analyzers/ folder